src/drt/src/serialization.h

```diff
@@ -562,6 +714,8 @@ void serialize_globals(Archive& ar)
   (ar) & VIAINPIN_TOPLAYER_NAME;
   (ar) & VIAINPIN_BOTTOMLAYERNUM;
   (ar) & VIAINPIN_TOPLAYERNUM;
+  // (ar) & VIAONLY_STDCELLPIN_BOTTOMLAYERNUM;
+  // (ar) & VIAONLY_STDCELLPIN_TOPLAYERNUM;
```

Comment:
Any reason not to serialize these?

Reply:
Not even used currently in tritonRoute. not sure why they are defined in the globals.

Reply:
Just remove it

Reply:
From globals as well? or just here?

Reply:
everywhere if unused.